package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -958,8 +958,8 @@ def generate_table(self):
         # build empty output table
         dtype = [
             ("time", float), ("donor_idx", int), ("acceptor_idx", int),
-            ("donor_resnm", "|S4"), ("donor_resid", int), ("donor_atom", "|S4"),
-            ("acceptor_resnm", "|S4"), ("acceptor_resid", int), ("acceptor_atom", "|S4"),
+            ("donor_resnm", "|U4"), ("donor_resid", int), ("donor_atom", "|U4"),
```

Comment:
On 29/01/16 22:53, kain88-de wrote:
>
> In package/MDAnalysis/analysis/hbonds/hbond_analysis.py 
> <https://github.com/MDAnalysis/mdanalysis/pull/683#discussion_r51321071>:
>
> > @@ -958,8 +958,8 @@ def generate_table(self):
> >          # build empty output table
> >          dtype = [
> >              ("time", float), ("donor_idx", int), ("acceptor_idx", int),
> > -            ("donor_resnm", "|S4"), ("donor_resid", int), ("donor_atom", "|S4"),
> > -            ("acceptor_resnm", "|S4"), ("acceptor_resid", int), ("acceptor_atom", "|S4"),
> > +            ("donor_resnm", "|U4"), ("donor_resid", int), ("donor_atom", "|U4"),
>
> So you are making everything unicode compatible with the Py3 fixes? Nice!
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MDAnalysis/mdanalysis/pull/683/files#r51321071>.
>
I try to. At least what breaks the tests...